src/nunavut/lang/properties.yaml

```diff
@@ -280,6 +280,27 @@ nunavut.lang.cpp:
     named_values:
         'true': 'true'
         'false': 'false'
+    comment_styles:
+        'zubax':
+            'prefix': ''
+            'comment': '/// '
+            'suffix': ''
```

Comment:
Shouldn't this one be called either `cpp` or `doxygen`? Documentation comments are usually different from regular comments.

Reply:
`//` is, by convention, referred to as "C++ style" and I've used it here as such. I've only ever seen `///` used for doxygen or by your Zubax style however doxygen style requires "at least two" lines using `///` so if we wanted to call this `cpp-doxygen` we'd need to add `///` as a prefix and a suffix. The problem with doing this is the DSDL comments are not structured so it wouldn't actually result in valid doxygen comments.

Reply:
>The problem with doing this is the DSDL comments are not structured so it wouldn't actually result in valid doxygen comments.

I don't quite follow, can you elaborate on why pasting DSDL docs may result in invalid Doxygen comments?

Reply:
well, actually, since the DSDL documentation is parsed or stropped there's no guarantee that it is valid documentation for any autodoc system. As such I've changed the names of things around to be about "block comments." If we ever want to parse the docs we can try to transform them into doxygen, sphinx-doc, etc; based on language.